src/composables/use-focus-on-show.js

```diff
@@ -35,26 +36,27 @@ export const useFocusOnShow = ({
      */
     ([dialog, visible, autoFocusOnShow, initialFocusElement]) => {
       if (!dialog || !visible || !autoFocusOnShow) return
-
-      const isActive = () => hasFocusWithin(dialog)
-
-      if (initialFocusElement) {
-        return ensureFocus(initialFocusElement, {
-          preventScroll: true,
-          isActive,
-        })
-      }
-
-      const tabbable = getFirstTabbableIn(dialog, true)
-
-      if (tabbable) {
-        ensureFocus(tabbable, { preventScroll: true, isActive })
-      } else {
-        ensureFocus(dialog, { preventScroll: true, isActive })
-        if (dialog.tabIndex === undefined || dialog.tabIndex < 0) {
-          warn(noFocusableElementWarning)
+      Vue.nextTick(() => {
```

Comment:
`nextTick` is available from the composition API. What is the reason to pull it off of `Vue` directly rather than importing it from `@nuxtjs/composition-api`?

Reply:
The main advantage of importing it from Vue is that when we (hopefully, some time in the future) move to Nuxt3, we won't need to update the imports. Other than that, there is no reason, really. Do you think it makes sense to use @nuxtjs/composition-api? At least for consistency with the current state of the code?

Reply:
I think for consistency. When (if) we move away from it, it will mostly be a find-and-replace job to update that import, right?